### PR TITLE
feature: Add again Custom_Scala_GetCalls

### PR DESCRIPTION
**Pull request by lolgab**
- Fix false negative when the get is wrapped in a call
  Example: `println(None.get)`

### CODE REVIEW

File: patterns-base/src/main/scala/codacy/patterns/Custom_Scala_GetCalls.scala [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package codacy.patterns
+
+import codacy.base.Pattern
+
+import scala.meta._
+
+case object Custom_Scala_GetCalls extends Pattern {
+
+  override def apply(tree: Tree) = {
```
Comment (rtfpessoa) [inline patterns-base/src/main/scala/codacy/patterns/Custom_Scala_GetCalls.scala:9]: Codacy found an issue: [Public method must have explicit type](https://app.codacy.com/gh/codacy/codacy-scalameta/pullRequest?prid=6180926)

File: patterns-base/src/main/scala/codacy/patterns/Custom_Scala_GetCalls.scala [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package codacy.patterns
+
+import codacy.base.Pattern
+
+import scala.meta._
+
+case object Custom_Scala_GetCalls extends Pattern {
```
Comment (rtfpessoa) [inline patterns-base/src/main/scala/codacy/patterns/Custom_Scala_GetCalls.scala:7]: Codacy found an issue: [Object name does not match the regular expression '^[A-Z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-scalameta/pullRequest?prid=6180926)

File: patterns-base/src/main/resources/patterns.json [hunk 3]
```diff
@@ -60,6 +60,12 @@
       "level": "Info",
       "enabled": true
     },
+    {
+      "patternId": "Custom_Scala_GetCalls",
+      "category": "ErrorProne",
+      "level": "Warning",
+      "enabled": true
```
Comment (franciscodua) [inline patterns-base/src/main/resources/patterns.json:67]: Do we want it enabled by default?
Given it will sometimes provide false positives

Comment (lolgab) [reply patterns-base/src/main/resources/patterns.json:67]: You're right, I'll put it as disabled

**Review by franciscodua — APPROVED** [review]

**Pull request merged**